.github/workflows/cart-ci.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
           fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
       - uses: ./.github/workflows/actions
       - name: Run Maven Build Command
-        run: mvn clean install -DskipTests -f cart
+        run: mvn clean install -DskipTests
```

Comment:
Please share us why we should remove "-f cart" here.

Reply:
When apply the saga framework I need to reuse models in the saga process, I added a project is 'saga', when apply the saga to any project I just add the saga dependency 
```
		<dependency>
			<groupId>com.yas</groupId>
			<artifactId>saga</artifactId>
			<version>0.0.1-SNAPSHOT</version>
			<scope>compile</scope>
		</dependency>
```
and you can see have a pom.xml in the root folder define modules, and when run the maven build I need to run with the file pom in root folder to build the 'saga' dependency used in order and cart project